src/storage/btree.c

```diff
@@ -11767,19 +11767,13 @@ btree_get_prefix_separator (const DB_VALUE * key1, const DB_VALUE * key2, DB_VAL
 {
   int c;
   int err = NO_ERROR;
-  static bool ignore_trailing_space = prm_get_bool_value (PRM_ID_IGNORE_TRAILING_SPACE);
```

Comment:
Is it not necessary to change the behavior according to the system parameters anymore?

Reply:
at btree_compare_key **do_coercion** is set to 2 and at **cmp_val ignore_trailing_space** is also checked.